### PR TITLE
Hungarian language update

### PR DESCRIPTION
**Pull request by ambrits**
I started translating the Hungarian language for the LMS

### CODE REVIEW

**Comment by michaelherger** [comment]
Wow! You've already done a ton of work - thanks a ton! Would you like me to merge already, or do you want to wait until you've got the plugins covered, too?

BTW: do you simply work your way through the strings file, line by line, or do you have a tool helping you with the work?

**Comment by ambrits** [comment]
Hi!For now, I'm just trying to figure out how the update works. This is my first github project :).  I am on vacation this week, I can continue the translation next week. I don't have a tool, I translate line by line in a string file.Tamás2024. aug. 11. dátummal, 8:29 időpontban Michael Herger ***@***.***> írta:﻿
Wow! You've already done a ton of work - thanks a ton! Would you like me to merge already, or do you want to wait until you've got the plugins covered, too?
BTW: do you simply work your way through the strings file, line by line, or do you have a tool helping you with the work?

**Comment by ambrits** [comment]
Hi! 

After the vacation, I will continue the translation. Is it possible to include what is ready so far so I can see it in action?

Best regards
Tamás

File: strings.txt [hunk 1]
```diff
@@ -373,6 +392,7 @@ SOFTSQUEEZE_HOME
 	FI	SoftSqueeze - koti
 	FR	Menu SoftSqueeze
 	HE	SoftSqueeze - ראשי
+	HU SoftSqueeze Home
```
Comment (michaelherger) [inline strings.txt:395]: Something's wrong here with the white space.

**Review by michaelherger — APPROVED** [review]
Well, it's impossible for me to review - I don't speak the language 😂. Let's give it a try. Thanks a ton!

**Comment by michaelherger** [comment]
Please open a new PR with any further addition.

**Comment by michaelherger** [comment]
BTW: there's a coverage chart on https://lyrion.org/contributing/adding-translations/. Hungarian has already overtaken some of the other languages 😉.

**Comment by ambrits** [comment]
Hi!

Thanks!

**Comment by ambrits** [comment]
Great :)

**Pull request merged**